### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by celestehorgan**
Signed-off-by: Celeste Horgan <celeste@cncf.io>

Fixes links mentioned in #437.

### CODE REVIEW

**Comment by yurishkuro** [comment]
Thanks, @celestehorgan .

If this fixes the links, would it make sense to add `make check-all-links` to `.travis.yml` in this PR to validate it?

```
script:
  - make check-all-links
  - bash ./travis/release.sh
```

**Comment by celestehorgan** [comment]
@yurishkuro I would rather do that in a separate PR, just to keep changes discrete.

This appears to work in so far as the link issue. I'm a little worried, though, because the [deploy preview's homepage](https://5f736db55a40cd00dd00f8ef--jaegertracing.netlify.app/) seems to have CSS issues (note that the subpages don't have the same issues, however).  But this is true of the [deploy for PR #439](https://5f6a06a2968d1b0008dee295--jaegertracing.netlify.app/) as well, so maybe this always happens for Jaeger's netlify preview builds?

I'm going to take the WIP off this PR and allow you to merge it, but let's keep an eye in case we break prod :)

**Comment by yurishkuro** [comment]
The previews used to work, not sure if something changes.

**Comment by yurishkuro** [comment]
The preview works now, but the URL is different https://deploy-preview-440--jaegertracing.netlify.app/

File: themes/jaeger-docs/layouts/docs/single.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
       <div class="column is-8 is-full-touch">
         <section class="content content--docs is-medium">
+          <p class="title is-1">Single.html</p>
```
Comment (yurishkuro) [inline themes/jaeger-docs/layouts/docs/single.html:17]: is this a debugging left-over?

Comment (yurishkuro) [reply themes/jaeger-docs/layouts/docs/single.html:17]: @celestehorgan should this line be removed?

Comment (celestehorgan) [reply themes/jaeger-docs/layouts/docs/single.html:17]: Yes – sorry for letting this go stale, it's been a busy few weeks :)

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
Thanks!

**Pull request merged**